src/Compilers/VisualBasic/Test/Semantic/Semantics/FieldInitializerBindingTests.vb

```diff
@@ -1408,6 +1408,77 @@ BC30799: Field 'Clazz.b' has an invalid constant value.
 </errors>)
         End Sub
 
+        <Fact, WorkItem(1028, "https://github.com/dotnet/roslyn/issues/1028")>
+        Sub WriteOfReadonlyStaticMemberOfAnotherInstatiation01()
```

Comment:
'Static' has a different meaning in VB. Consider renaming to `WriteOfReadonlySharedMemberOfAnotherInstantiation01`